gaia-fast-list.js

```diff
@@ -450,7 +450,7 @@ var GaiaFastListProto = {
 
       [picker] .picker {
         position: absolute;
-        offset-inline-end: 0;
+        right: 0;
```

Comment:
I don't know anything about the picker, but this line seems suspect. Are you sure it's correct? If so, then great! I just wanted to be sure.